docs/rackhd/workflow_progress_notification.rst

```diff
@@ -119,16 +119,42 @@ Besides task started and finished events, a time-consuming task is not proper to
+Milestone name      Milestone description
+=================== ==============================================================================================
+requestProfile      Enter ipxe and request OS installation profile. Common milestone for all OSes.
+enterProfile        Enter profile, start to download installer. Common milestone for all OSes.
+startInstaller      Start installer and prepare installation. Common milestone for all OSes.
+preConfig           Enter Pre OS configuration.
+startSetup          Net use Windows Server 2012 and start setup.exe. Only used for Windows Server.
+installToDisk       Execute OS installation. Only used for CoreOS.
+startPartition      Start partition. Only used for Ubuntu.
+postPartitioning    Finished partitioning and mounting, start package installation. Only used for SUSE.
+chroot              Finished package installation, start first boot. Only used for SUSE.
+postConfig          Enter Post OS configuration.
+completed           Finished OS installation. Common milestone for all OSes.
+=================== ==============================================================================================
+
+Below table includes default milestone sequence for RackHD supported OSes:
+
+=============== =================== ==============================================================================================
+OS Name         Milestone Quantity  Milestones in Sequence
+=============== =================== ==============================================================================================
+CentOS, RHEL    6                   1.requestProfile; 2.enterProfile; 3.startInstaller; 4.preConfig; 5.postConfig; 6.completed
+Esxi            6                   1.requestProfile; 2.enterProfile; 3.startInstaller; 4.preConfig; 5.postConfig; 6.completed
+CoreOS          5                   1.requestProfile; 2.enterProfile; 3.startInstaller; 4.installToDisk; 5.completed
+Ubuntu          7                   1.requestProfile; 2.enterProfile; 3.startInstaller; 4.preConfig; 5.startPartition; 6.postConfig; 7.completed
+WindowServer    5                   1.requestProfile; 2.enterProfile; 3.startInstaller; 4.startSetup; 5.completed
+SUSE            7                   1.requestProfile; 2.enterProfile; 3.startInstaller; 4.preConfig; 5.postPartitioning; 6.chroot; 7.completed
+PhotonOS        5                   1.requestProfile; 2.enterProfile; 3.startInstaller; 4.postConfig; 5.completed
+=============== =================== ==============================================================================================
+
+In progress message, milestone quantity will be set to `maximum` and sequence number to `value` while RackHD is installing OS.
+Take CentOS as an example, with completion of `postConfig` milestone, RackHD will send progress information to user with `value` 5 and `maximum` 6. `value` varies from 1 to 5 and with `maximum` fixed 6 during whole CentOS installation progress.
```

Comment:
it might be more clear to add an example here for user's better understanding.
```
{
    ...,
    taskProgress: {
        taskId: '...',
        taskName: 'Install CentOS',
        progress: {
            value: 5,
            maximum: 6,
            description:  ???,
            percentage: ???
        }
    }
}
```